apps/ui/src/components/FormValidation.vue

```diff
@@ -48,7 +50,7 @@ function handleStrategySave(value: Record<string, any>) {
   };
 }
 
-onMounted(() => {
+watchEffect(() => {
```

Comment:
Maybe better to use `computed`? instead of `watchEffect` and `ref`

```ts
const votingStrategies = computed(() => model.value?.params?.strategies || []);
```

Reply:
Had to use computed with get/set for it, because we also write to it.